src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+abstract public class Task {
+    private final String name;
+    private boolean done;
```

Comment:
Changing to isDone would have been better to follow the standard? For other boolean variables as well!